tests/integration/features/features_suite_int_test.go

```diff
@@ -71,6 +72,9 @@ var _ = BeforeSuite(func() {
 	Expect(err).NotTo(HaveOccurred())
 	Expect(config).NotTo(BeNil())
 
+	err = featurev1.AddToScheme(testScheme)
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
We should move it next to L56 and make it look like:

```golang
	utilruntime.Must(featurev1.AddToScheme(testScheme))
```

Reply:
[9455151](https://github.com/opendatahub-io/opendatahub-operator/pull/801/commits/94551518ce359dce2f402dc0b865accad940aeba)